test-utils/src/main/scala/com/mesosphere/utils/mesos/MesosTest.scala

```diff
@@ -338,7 +338,9 @@ case class MesosCluster(
         s"--port=$port",
         s"--resources=${resources.resourceString()}",
         s"--master=$masterUrl",
-        s"--work_dir=${workDir.getAbsolutePath}") ++ extraArgs,
+        s"--work_dir=${workDir.getAbsolutePath}",
+        s"--cgroups_root=mesos$port", // See MESOS-9960 for more info
+        s"""--executor_environment_variables={"GLOG_v": "2"}""") ++ extraArgs,
```

Comment:
This particular line was needed to test executor failures in Marathon. Not sure if you want to have it active or just commented out for the rainy day.

Reply:
Is it super noisy to enable? Let's comment it out if so